lib/bolt/outputter/human.rb

```diff
@@ -532,41 +529,54 @@ def print_target_info(target_list, inventoryfile)
           info << indent(2, "No targets\n\n")
         end
 
-        @stream.puts info
+        info << format_inventory_source(inventoryfile)
+        info << format_target_summary(inventory_targets.count, adhoc_targets.count)
 
-        print_inventory_summary(
-          inventory_targets.count,
-          adhoc_targets.count,
-          inventoryfile
-        )
+        @stream.puts info
       end
 
-      private def print_inventory_summary(inventory_count, adhoc_count, inventoryfile)
+      private def format_inventory_source(inventoryfile)
         info = +''
 
         # Add inventory file source
-        info << colorize(:cyan, "Inventory file\n")
-        info << if File.exist?(inventoryfile)
+        info << colorize(:cyan, "Inventory source\n")
+        info << if ENV[Bolt::Inventory::ENVIRONMENT_VAR]
+                  indent(2, "#{Bolt::Inventory::ENVIRONMENT_VAR} environment variable\n")
+                elsif File.exist?(inventoryfile)
                   indent(2, "#{inventoryfile}\n")
                 else
                   indent(2, wrap("Tried to load inventory from #{inventoryfile}, but the file does not exist\n"))
```

Comment:
In what situation is this not already caught when we load the inventory in the CLI?

Reply:
Can you clarify? Not quite sure what you're asking here.

Reply:
Ah, sorry, I could definitely have asked more clearly! I was wondering if this was dead code because we check for inventory file existence when loading from the CLI, but it's not because this gets hit when Bolt falls back to the default inventory file and it doesn't exist :+1: